spec/models/miq_request_workflow_spec.rb

```diff
@@ -535,4 +535,64 @@
       expect(workflow.storage_to_hash_struct(storage).storage_clusters).to be_nil
     end
   end
+
+  context "#password_helper encrypts and decrypts the correct fields" do
+    before do
+      MiqRegion.seed
+      # Sorry for allow_any_instance_of, but I don't want to load the automate model, it's not needed for the test
```

Comment:
@chrisarcand If you have ideas on how to avoid this (without loading the automate model) I'm all ears.
The ideas I came up with all sound worse:
- Prepend the class to stub `get_dialogs` and `update_field_visibility`
- Leverage `allocate` to get in before `initialize` and `expect` the instance to receive `get_dialogs`, etc.
But at least this `allow_any_instance_of` points out that `initialize` may be doing more than it should and we need some refactoring.